### PR TITLE
adding create multipart upload permission

### PR DESCRIPTION
**Pull request by aibitayo55555**

### CODE REVIEW

File: policy.tf [hunk 1]
```diff
@@ -311,6 +315,7 @@ data "aws_iam_policy_document" "s3_bucket_policy_document_whitelist" {
       "s3:ListMultipartUploadParts",
       "s3:PutObjectAcl",
       "s3:RestoreObject",
+      "s3:CreateMultipartUpload",
```
Comment (vinn946) [inline policy.tf]: Duplicate entry - remove

File: policy.tf [hunk 2]
```diff
@@ -429,6 +435,7 @@ data "aws_iam_policy_document" "s3_bucket_with_kms_and_whitelist_vpc_policy_docu
...
```
Comment (vinn946) [inline policy.tf]: Duplicate entry - remove

File: policy.tf [hunk 3]
```diff
@@ -711,6 +723,7 @@ data "aws_iam_policy_document" "s3_bucket_with_kms_and_whitelist_ip_and_vpc_poli
...
```
Comment (vinn946) [inline policy.tf]: Duplicate entry - remove

File: policy.tf [hunk 4]
```diff
@@ -645,6 +655,7 @@ data "aws_iam_policy_document" "s3_bucket_with_kms_and_whitelist_ip_and_vpc_poli
...
```
Comment (vinn946) [inline policy.tf]: Duplicate entry - remove

File: policy.tf [hunk 5]
```diff
@@ -527,6 +535,7 @@ data "aws_iam_policy_document" "s3_bucket_with_whitelist_vpc_policy_document" {
...
```
Comment (vinn946) [inline policy.tf]: Duplicate entry - remove

File: policy.tf (hunk 2 above)
Comment (tasharnvb) [inline policy.tf]: Duplicate line

File: policy.tf (hunk 5 above)
Comment (tasharnvb) [inline policy.tf]: Same as above

File: policy.tf (hunk 4 above)
Comment (tasharnvb) [inline policy.tf]: Same as above

File: policy.tf (hunk 3 above)
Comment (tasharnvb) [inline policy.tf]: Same as above

**Review by vinn946 — CHANGES_REQUESTED** [review]
Action's been added multiple times in the same IAM policy document
Otherwise LG

**Review by vinn946 — APPROVED** [review]
LG

**Pull request merged**